ci/cirrus-templates/script.bash

```diff
@@ -14,7 +14,7 @@ echo "========="
 echo "Display the current git status"
 
 git status
-git describe
+git describe --always
```

Comment:
Wow, moving the FreeBSD Ci to GitHub Actions is surprisingly doable! The only problem is that `git describe` will fail (the change to `git describe --always` should be reverted).

Reply:
Weirdly enough, what worked yesterday shows the following today:

```console
+ git fetch --prune --unshallow --tags
  fatal: shallow file has changed since we read it
```

Reply:
I have finally solved this issue by fetching the tags outside of FreeBSD.